.gitattributes

```diff
@@ -1 +1,3 @@
 * text=auto
+* text  eol=lf
```

Comment:
This is not a good way to specify it. It forces all files to be normalized, even binary. It also makes the line above it moot.

Instead, specify explicitly the file types to LF normalize.